web-react-ts/src/permission-utils.ts

```diff
@@ -164,7 +164,7 @@ export const permitArrivals = (churchLevel: ChurchLevel): Role[] => {
       break
   }
 
-  if (churchLevel !== 'Oversight') {
+  if (churchLevel === 'Stream') {
     return [...permitAdmin(churchLevel), ...permittedFor]
```

Comment:
You need to change the type def on line 180 to reflect that onlly the word 'Stream' can be used